mesh_client/__init__.py

```diff
@@ -318,7 +411,8 @@ def ping(self) -> dict:
 
     def handshake(self):
```

Comment:
no return type - same for a bunch of other methods - well it was. There's still iterate_all_messages() and iterate_messages(). I've just had to refresh so I think you may have just added some return types :-)

Reply:
handshake not expected to return anything, so think this is correct